rskj-core/src/main/java/co/rsk/remasc/Remasc.java

```diff
@@ -199,7 +202,19 @@ RskAddress getRskLabsAddress() {
     }
 
     private Coin payToFederation(Constants constants, boolean isRskip85Enabled, Block processingBlock, BlockHeader processingBlockHeader, Coin syntheticReward) {
-        RemascFederationProvider federationProvider = new RemascFederationProvider(activationConfig, constants.getBridgeConstants(), repository, processingBlock);
+
+        ActivationConfig.ForBlock activations = activationConfig.forBlock(processingBlock.getNumber());
```

Comment:
Since we already have the activations object, does it makes to receive a `boolean isRskip85Enabled`? 

Totally out of scope for this task but could you please have a look and see if it's something redundant or if there is a reason for this?

Reply:
Updated!